proxy.go

```diff
@@ -18,7 +18,9 @@ func writeRegistryAndFlushToProxy(c *WavefrontConfig) error {
 	}
 	defer conn.Close()
 	w := bufio.NewWriter(conn)
-
+	if c.Prefix != "" {
+		c.Prefix = strings.Join([]string{c.Prefix, "."}, "")
```

Comment:
Can you please improve the comment for the Prefix variable in the `WavefrontConfig` to indicate it should not end with a period.

Reply:
TFTR, updated the comments for WavefrontConfig